litmus-portal/frontend/src/components/ProgressBar/AnalyticsLinearProgressBar/styles.tsx

```diff
@@ -0,0 +1,19 @@
+import { makeStyles, Theme } from '@material-ui/core/styles';
```

Comment:
Please change the extension to .ts like we have for other style files @bansalsurya